### PR TITLE
Now we can see the review text when writing

### PR DESCRIPTION
**Pull request by sgueissa**
It's also fix the fact that the app was crashing when opening the reviews when there was no more Quotas for the Youtube API :(

### CODE REVIEW

File: .cirrus.yml [hunk 1]
```diff
@@ -26,6 +26,8 @@ check_android_task:
     -no-snapshot
     -no-window
     -camera-back none
+    -skin 1080x1920
```
Comment (MariemBaccari) [inline .cirrus.yml:29]: Thanks, we really needed that :)

File: app/src/main/java/com/github/sdp/mediato/ui/NewItemFragment.java [hunk 2]
```diff
@@ -31,51 +33,69 @@
...
     }
 
+    private void setObserverOnKeyboard(View view) {
+        // Get references to the views
+        ScrollView scrollView = view.findViewById(R.id.scrollView);
+
+        // Set an OnLayoutChangeListener to the root view to detect layout changes
+        scrollView.getViewTreeObserver().addOnGlobalLayoutListener(new ViewTreeObserver.OnGlobalLayoutListener() {
+            @Override
+            public void onGlobalLayout() {
```
Comment (MariemBaccari) [inline app/src/main/java/com/github/sdp/mediato/ui/NewItemFragment.java:86]: I'm not sure I understand what these calculations are for. Can you explain ?

File: app/src/main/java/com/github/sdp/mediato/ui/NewItemFragment.java [hunk 3]
```diff
@@ -125,14 +145,19 @@ private void searchTrailer() {
...
+                                    handleTrailerResponse(listResponse.getItems().get(0)));
+                }
             } catch (IOException e) {
-                throw new RuntimeException(e);
-            }
-            if (getActivity() != null && listResponse != null && !listResponse.getItems().isEmpty()) {
-                getActivity()
-                        .runOnUiThread(() -> // we go back to ui thread (mandatory)
-                                handleTrailerResponse(listResponse.getItems().get(0)));
+                getActivity().runOnUiThread(() -> {
```
Comment (MariemBaccari) [inline app/src/main/java/com/github/sdp/mediato/ui/NewItemFragment.java:154]: Good catch here !

**Review by MariemBaccari — APPROVED** [review]
Good job and thanks for fixing the youtube bug that makes the app unecessarily crash ! I just have one question but the PR is clean.

File: app/src/main/java/com/github/sdp/mediato/ui/NewItemFragment.java (hunk 2 above)
Comment (sgueissa) [reply app/src/main/java/com/github/sdp/mediato/ui/NewItemFragment.java:86]: it just to detect when the keyboard open and scroll down to the bottom to see the review text and the button add

**Pull request merged**